spec/models/stamp_spec.rb

```diff
@@ -15,6 +15,12 @@
     it 'is not valid without an owner_id' do
       expect(build(:stamp, owner_id: nil)).not_to be_valid
     end
+
+    it 'is valid with a name unique to only active stamps' do
+      stamp = create(:stamp)
+      stamp.destroy!
```

Comment:
I think this needs to be `stamp.deactivate!` otherwise it's testing you build a Stamp with the same name as one that has been destroyed.